commercetools/resource_project.go

```diff
@@ -232,6 +253,28 @@ func projectUpdate(d *schema.ResourceData, client *commercetools.Client, version
 		}
 	}
 
+	if d.HasChange("carts") {
```

Comment:
Does this path handle deleting the property correctly (I assume it would set it to false?)

Reply:
Tests contain an explicit `resource.TestCheckNoResourceAttr` on removing the `carts` block. So it looks to be working. Could do an explicit 
```go
else {
			input.Actions = append(input.Actions, &commercetools.ProjectChangeCountryTaxRateFallbackEnabledAction{CountryTaxFallbackEnabled: nil})
		}
```

To be sure. 

Same goes for the messages block. See also the comment I added there. I've got a bad feeling there's some implicit functionality somewhere in this I'm missing.

Reply:
Added explicit set to False actions for delete (of both Messages and Carts)